src/main/java/com/questhelper/quests/thefeud/TheFeud.java

```diff
@@ -1,27 +1,3 @@
-/*
```

Comment:
You need to keep the original copyright, and add yourself to it as this is a substantial re-write.

Reply:
Oh sorry, I had started the file again from scratch and forgot about the Copyright notice at the top cause intellij collapses it!

Reply:
Updated with b5838de